src/app/pages/select-network/networks-drawer.tsx

```diff
@@ -0,0 +1,43 @@
+import { useNavigate } from 'react-router-dom';
+
+import { RouteUrls } from '@shared/route-urls';
+import { NetworkListLayout } from '@app/pages/select-network/components/network-list.layout';
+import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
+import { BaseDrawer } from '@app/components/drawer/base-drawer';
+import { NetworkListItem } from '@app/pages/select-network/network-list-item';
+import { useNetworks } from '@app/store/networks/networks.selectors';
+import { AddNetworkButton } from './components/add-network-button';
+import { useNetworksActions } from '@app/store/networks/networks.hooks';
+
+export function SelectNetwork() {
+  const navigate = useNavigate();
+  const networks = useNetworks();
+  const analytics = useAnalytics();
+  const networksActions = useNetworksActions();
+
+  function addNetwork() {
+    void analytics.track('add_network');
+    navigate(RouteUrls.AddNetwork);
+  }
+
+  function selectNetwork(id: string) {
+    void analytics.track('change_network', { id });
+    networksActions.changeNetwork(id);
+    navigate('..');
+  }
+
+  function closeNetworkModal() {
+    navigate('..', { replace: true });
```

Comment:
What makes this call to `navigate` different from the others to use `replace: true`?

Reply:
This isn't necessary actually, good spot, removed